docs/Cards.html

```diff
@@ -0,0 +1,477 @@
+                <p class=" header-6 header-6 content__desc">
+                    The inputs also have 3 States shown below which can be
+                    toggled by using <i class="code--blue">---tonal</i> along with the respective states using <i
+                        class="code--blue">---success</i>, <i class="code--blue">---warning</i>
+                    and <i class="code--blue">---primary</i>.</p>
+                <!-- Make the component wrapper in row or column accordingly -->
+                <div class="component__wrapper--column bg-blue border-8d  gap20 text--left">
+
+                    <div class="component__item--row">
+                        <!-- Insert Component Here -->
+                        <!-- Cards With Dismiss -->
+                        <div class="card card--small box-shadow">
+                            <div class="card__img--small image--responsive">
+                                <img src="https://picsum.photos/id/684/400/400">
+                            </div>
+                            <div class="card__content--column ">
+                                <p class="subtitle-1 pad-12side ">Smartphones</p>
+                                <p class="body-2 pad-12side">Starting from 10k</p>
+                            </div>
+                            <span class="card__icon material-icons">close</span>
+                        </div>
+                        <!-- Insert Component Label Here -->
+                        <span class="subtitle-1">Cards with Dismiss</span>
+                    </div>
+
+
+
+                </div>
+            </section>
+            <!-- Implementation Code Section -->
+            <section class="iris__desc__wrapper">
+                <h2 class="header-h2 content__title">Implementation</h2>
+                <p class=" header-6 content__desc">
+                    Not everyone likes to be round and perfect. Some prefer cute corners!</p>
+                <!-- Make the component wrapper in row or column accordingly -->
+                <div class="code__wrapper code__wrapper code--blue gap20">
+                    <pre><code>    <!-- Cards With Dismiss -->
+                        <div class="card card--small box-shadow">
+                            <div class="card__img--small image--responsive">
+                                <img src="https://picsum.photos/id/684/400/400">
+                            </div>
+                            <div class="card__content--column ">
+                                <p class="subtitle-1 pad-12side ">Smartphones</p>
+                                <p class="body-2 pad-12side">Starting from 10k</p>
+                            </div>
+                            <span class="card__icon material-icons">close</span>
+                        </div></code></pre>
+                </div>
+
+
+            </section>
+            <!-- Page Navigation -->
+            <section class="iris__desc__wrapper page_nav">
```

Comment:
use nav tag for navigation bars

Reply:
Resolved in the latest commit. Thank You.